api_tests/users/views/test_user_external_identities.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from osf_tests.factories import AuthUserFactory
+from api.base.settings.defaults import API_BASE
+
+
+@pytest.fixture()
+def user():
+    user = AuthUserFactory()
+    user.external_identity = {
+        'ORCID': {
+            '0000-0001-9143-4653': 'VERIFIED'
+        },
+        'LOTUS': {
+            '0000-0001-9143-4652': 'LINK'
+        }
+    }
+    user.save()
+    return user
+
+
+@pytest.fixture()
+def unauthorized_user():
+    return AuthUserFactory()
+
+
+@pytest.mark.django_db
+class TestUserIdentitiesList:
+
+    @pytest.fixture()
+    def url(self, user):
+        return '/{}users/{}/settings/identities/'.format(API_BASE, user._id)
+
+    def test_authorized_gets_200(self, app, user, url):
+        res = app.get(url, auth=user.auth)
+        print res.json['data']
+        assert res.status_code == 200
+        assert res.content_type == 'application/vnd.api+json'
+        assert res.json['data'][0]['attributes'] == {u'status': u'LINK', u'external_id': u'0000-0001-9143-4652'}
```

Comment:
You could define data = res.json['data']. Also, for all these tests, I prefer comparing individual attributes instead of comparing dictionaries for readability.

assert data[0]['attributes']['status'] == 'LINK'